interface/ceed-operator.c

```diff
@@ -1804,9 +1852,11 @@ int CeedOperatorDestroy(CeedOperator *op) {
   CeedCall(CeedQFunctionDestroy(&(*op)->dqf));
   CeedCall(CeedQFunctionDestroy(&(*op)->dqfT));
   // Destroy any composite labels
-  for (CeedInt i = 0; i < (*op)->num_context_labels; i++) {
-    CeedCall(CeedFree(&(*op)->context_labels[i]->sub_labels));
-    CeedCall(CeedFree(&(*op)->context_labels[i]));
+  if ((*op)->is_composite) {
```

Comment:
Not sure why this would only apply to composite operators. Non-composite operators won't have labels with sub_labels, but they could have multiple labels, right?

Reply:
It only applies to composite operators because non-composite labels are owned by the QFunctionContext

Reply:
Ah, gotcha. Makes sense.